control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -3861,6 +3861,90 @@ def test_service_phase21_readiness_surfaces_optional_extension_operability_defau
+        store, _ = support.make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",  # noqa: S106 - test fixture secret
+                wazuh_ingest_reverse_proxy_secret="reviewed-proxy-secret",  # noqa: S106 - test fixture secret
+                admin_bootstrap_token="reviewed-admin-bootstrap-token",  # noqa: S106 - test fixture secret
+            ),
+            store=store,
+        )
+        service.persist_record(
+            AITraceRecord(
```

Ruff (0.15.12)

[error] Undefined name `AITraceRecord`

(F821)